features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/item-manage/template.jag

```diff
@@ -261,7 +263,55 @@
                             </div>
                         </div>
                     </div>
-
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <h4 class="panel-title">
+                                <div>
+                                    <div class="heading no-description"><%=i18n.localize("API Products")%></div>
+                                </div>
+                            </h4>
+                        </div>
+                        <div class="panel-collapse">
+                            <div class="panel-body form-group">
+                                <label class="control-label col-sm-3"><%=i18n.localize("Select an option")%>:</label>
+                                <div class="col-sm-8">
+                                    <label class="radio" for="bundleProduct">
+                                       <input type="radio" class="product_options" id ="bundleProduct" name="product_options"
+                                       checked="checked" value="bundleProduct">
+                                       <span class="helper"><%=i18n.localize("Bundle with an existing API Product")%></span>
+                                    </label><br>
+                                    <div class="form-group productList"><br>
+                                        <label class="control-label col-xs-3"><%=i18n.localize("API Products")%>:
+                                        </label>
+                                        <div class="col-lg-5">
```

Comment:
Fix is in https://github.com/dewmini/carbon-apimgt/commit/31e5bed7b95384e6753051bd58a25cf3d1abbd01